l10n/en/firefox/privacy/book.ftl

```diff
@@ -0,0 +1,130 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/privacy/book
+
+# HTML page title
+little-book-of = Little Book of Privacy
```

Comment:
Please prefix Fluent IDs with a name space.

Reply:
Is this an ID? It's only used once in the doc. Perhaps I don't understand.

Reply:
All Fluent string IDs in this file should be namespaced with a unique prefix (as per the docs and in other files), and just like you have done in #9586

For example: 

```
privacy-book-little-book-of = Little Book of Privacy
privacy-book-how-to-stay = How to stay relaxed and be secure online.
```

Right now there's no namespacing, which means it's hard to work out what file a string belongs to in the template, and also increases the likelihood of creating duplicate IDs over time.